client/agora/views/dashboard/partials/gallery/note-gallery.ejs

```diff
@@ -82,43 +83,100 @@
-
-                <% }else if(i % 3===0) {%>
-              </div>
-              <div class="row ">
-                <% renderCard(i-1,availableResources[i-1].resourceName,availableResources[i-1].resourceDescription,["Bio","Chem"],"Goal", ["Doc 1","Doc 2","Doc 3", "Doc 4","Doc 5"])
-                  %>
-
-
-                  <% }else{ %>
+                  <% }else{ 
+                    if(i % 2 === 0) {%>
                     <% renderCard(i,availableResources[i-1].resourceName,availableResources[i-1].resourceDescription,["Bio","Chem"],"Note", null)
-                      %>
+                    }else{
+                      renderCard(i-1,availableResources[i-1].resourceName,availableResources[i-1].resourceDescription,["Bio","Chem"],"Goal", ["Doc 1","Doc 2","Doc 3", "Doc 4","Doc 5"])
+                     } %>
                       <% }}%>
               </div>
+            
 
 
 
              
 
                 <script>
+                  var toggleGoalView = () => {
+                 const input = document.querySelector(".form-control");
+                  const cards = document.getElementsByClassName("col");
+                  console.log(cards[0])
+                for (let i = 0; i < cards.length; i++) {
+                let title = cards[i].querySelector(".card-body");
+                if(title.id === "add" ){
+                  cards[i].classList.remove("d-none");
+                }
+                else if (title.id === 'goal' ) {
+                    cards[i].classList.remove("d-none")
+                } else {
+                    cards[i].classList.add("d-none")
+                }
+            }
+        } 
+        var toggleTopicView = () => {
+                 const input = document.querySelector(".form-control");
+                  const cards = document.getElementsByClassName("col");
+                  console.log(cards[0])
+                let filter = input.value
+                for (let i = 1; i < cards.length; i++) {
+                let title = cards[i].querySelector(".card-body");
+                console.log(title.id)
+                if(title.id === "add" ){
+                  cards[i].classList.remove("d-none");
+                }
+                else if (title.id === 'topic' ) {
+                    cards[i].classList.remove("d-none")
+                } else {
+                    cards[i].classList.add("d-none")
+                }
+            }
+        }
+        var toggleAllView = () => {
+                 const input = document.querySelector(".form-control");
+                  const cards = document.getElementsByClassName("col");
+                  console.log(cards[0])
+                let filter = input.value
+                for (let i = 0; i < cards.length; i++) {
+                let title = cards[i].querySelector(".card-body");
+                    cards[i].classList.remove("d-none")
+                
+            }
+        }
                   $('.dropdown-content').on('click', function (ev) {
```

Comment:
Is this jquery?? I would prefer to keep the JS vanilla! 
There is no reason to use a library when you can simply write:
document.getElementbyClassName('dropdown-content').addEventListener('click', (ev) => { ... });